packages/office-ui-fabric-react/src/components/ScreenReaderAlert/examples/ScreenReaderAlert.After.Example.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { ScreenReaderAlert } from 'office-ui-fabric-react/lib/ScreenReaderAlert';
+import { PrimaryButton, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+
+const alertMessage: string = `This message should be read after the content of button B.`;
+
+export interface IScreenReaderAlertAfterExampleState {
+  screenReaderIndicator?: number;
+}
+
+export class ScreenReaderAlertAfterExample extends React.Component<any, IScreenReaderAlertAfterExampleState> {
+  private _lateFocusedButton: HTMLButtonElement;
+
+  constructor(props: any) {
+    super(props);
+
+    this.state = {
+      screenReaderIndicator: 0
+    };
+
+    this._onButtonClicked = this._onButtonClicked.bind(this);
+  }
+
+  public render() {
+    return (
+      <div>
+        <ScreenReaderAlert
+          indicator={ this.state.screenReaderIndicator }
+          text={ alertMessage }
+        />
+        <PrimaryButton onClick={ this._onButtonClicked }>
+          { 'Button A: Trigger to move focus to button B and read message' }
+        </PrimaryButton>
+        <DefaultButton
+          ref={ (instance) => this._lateFocusedButton = ReactDOM.findDOMNode(instance) as HTMLButtonElement }
```

Comment:
Can you update this:

```
<DefaultButton
  componentRef={ b => this._lateFocusedButton = b }
  ...
/>
```

This returns `IButton`, which has focus method. You should avoid using findDOMNode, as it encourages people to reach into private assumptions about a component.